jenkins/shared-library/vars/runScript.groovy

```diff
@@ -6,7 +6,7 @@ void call(String scriptPath, String filename) {
 }
 
 void call(String scriptFilepath) {
-	call(scriptFilepath, scriptFilepath, true)
+	call(scriptFilepath, scriptFilepath, false)
```

Comment:
why this change?

Reply:
some logging was missing in Jenkins logs.